graphql/resolvers/tasks.js

```diff
@@ -0,0 +1,42 @@
+const Task = require("../../models/task");
+const moment = require("moment");
+
+module.exports = {
+  tasks: async (args) => {
+    try {
+      let tasks = await Task.find({ userId: args.userId }).populate(
+        "assignedBy"
+      );
+      return tasks.map((result) => {
+        return {
+          ...result._doc,
+          date: moment(result._doc.date).format("ll"),
+          dueDate: moment(result._doc.dueDate).format("ll"),
+        };
+      });
+    } catch (err) {
+      throw err;
+    }
+  },
+  createTasks: async (args, req) => {
```

Comment:
Can't understand usecase of this endpoint since a Task can be created only from a Message instance; better to handle it via sockets only